styles/globals.css

```diff
@@ -1645,3 +1645,53 @@ select {
     margin-top: 10px;
   }
 }
+
+
+/*////////*/
+@media(max-width:768px) {
+  .about-section{
+    flex-direction: column;
+  }
+  .about{
+    margin-bottom: 68px;
+    left: 100px;
+  }
+  .mentor-div1{
+    padding: 12px;
+    margin-top: 20px;
+    margin-right: 81px;
+    padding-bottom: 42px;
+    border-bottom: 2px solid grey;
+   margin-left: 12px;
+   margin-right: 12px;
+}
+.mentor-div2{
+  justify-content: center;
+  top: 0px;
+  margin-top: 52px;
+  margin-left: 104px;
```

Comment:
move it to mentorProfile css file.. as it is not global property!

Reply:
Done👍